frontend/conf/routes

```diff
@@ -91,6 +91,7 @@ GET            /patrons                               controllers.Info.patron
 GET            /supporter                             controllers.Info.supporter
 GET            /us/supporter                          controllers.Info.supporterUSA
 GET            /eu/supporter                          controllers.Info.supporterEurope
+GET            /intl/supporter                        controllers.Info.supporterInternational
```

Comment:
http://www.theguardian.com/preference/edition/int

As a general rule, we tried to map the country groups to what we had on the main website, that is us, uk, au and int, that's why the CountryGroup ids are exactly those.

But I guess that's not the end of the world, it's just a route. It's just a bit confusing, as the associated countryGroup query parameter is `int` instead of `intl`